### PR TITLE
Add mixlib-shellout, terminal-table and thor packages

### PR DESCRIPTION
**Pull request by xprazak2**
These are required for jenkins_api_client

### CODE REVIEW

**Comment by ehelms** [comment]
I believe thor is already packaged in the SCL -- https://www.softwarecollections.org/repos/rhscl/ruby193/epel-7-x86_64/

File: rel-eng/tito.props [hunk 1]
```diff
@@ -18,10 +18,16 @@ whitelist = katello-repos katello-agent
...
+whitelist = rubygem-mixlib-shellout
+  rubygem-terminal-table
+  rubygem-thor
 
 [katello-nightly-rhel7]
 disttag = .el7
 scl = tfm
+whitelist = rubygem-mixlib-shellout
+  rubygem-terminal-table
+  rubygem-thor
```
Comment (ehelms) [inline rel-eng/tito.props]: Why do these need to be whitelisted here?

**Comment by xprazak2** [comment]
@ehelms: I removed the whitelisting, it should get packaged as 'tfm' even without it, right? I believe I need thor in tfm as well...

**Comment by ehelms** [comment]
As long Thor is available in an SCL that we use then we don't need to package it. Currently it is available in ruby193 and rh-ruby22.

**Comment by xprazak2** [comment]
OK, I removed thor.

**Comment by ehelms** [comment]
ACK - merging and will build and re-test the next one in line

**Pull request merged**